src/main/java/rife/engine/CookieBuilder.java

```diff
@@ -43,17 +43,6 @@ public Cookie cookie() {
         return cookie_;
     }
 
-    /**
```

Comment:
Why did you remove the comment and version methods in the builder?

Reply:
Deprecated.

Reply:
It's still part of the API and since the builder forwards this, I think we should keep it in there

Reply:
IT won't be there in the next API version, so I don't think we should.

Reply:
To be clear, it's [deprecated and scheduled for removal](https://jakarta.ee/specifications/servlet/6.0/apidocs/jakarta.servlet/jakarta/servlet/http/cookie#setComment(java.lang.String)).